tools/driver/driver.cpp

```diff
@@ -180,6 +185,27 @@ static int run_driver(StringRef ExecName,
 }
 
 int main(int argc_, const char **argv_) {
+#ifdef _WIN32
```

Comment:
Windows still doesn't print out the command line arguments at all, response files or not. From what I can tell, llvm uses Backtrace to do it, and does nothing if it can't find it, so I'll have to find a different way on Windows.